west.yml

```diff
@@ -240,7 +240,7 @@ manifest:
       groups:
         - hal
     - name: hal_stm32
-      revision: 02d82ee3aa0daa29ab6f107f9969508b89bc6bf7
+      revision: pull/282/head
```

Comment:
PR has been merged, please update this.
(You'll probably need a rebase before that)